onnxruntime/core/session/inference_session.cc

```diff
@@ -911,9 +928,8 @@ Status InferenceSession::LoadOrtModel(std::function<Status()> load_ort_format_mo
   // Check version mismatch, for now we will only proceed when runtime version matches the model's ort version
   const auto* fbs_ort_version = fbs_session->ort_version();
   ORT_RETURN_IF(fbs_ort_version == nullptr, "Serialized version info is null. Invalid ORT format model.");
-  ORT_RETURN_IF_NOT(fbs_ort_version->str() == ORT_VERSION,
-                    "ORT_VERSION mismatch. Saved model ORT version: ", fbs_ort_version->str(),
-                    ", Current ORT version: ", ORT_VERSION);
+  ORT_RETURN_IF_NOT(IsOrtModelVersionSupported(fbs_ort_version->str()),
+                    "The ort model version [", fbs_ort_version->str(), "] is not supported in current onnxruntime");
 
```

Comment:
nit: 
ort -> ORT. 
'in current onnxruntime' -> 'in this build' for consistency with other messaging.